core/scripts/local/ConvertWeth.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * @notice Wrap or unwrap --collateral amount of WETH. Unwrap by passing in `--unwrap true` flag.
```

Comment:
nit: I think `--unwrap` should work without the `true`, right?

Reply:
Correct!